eralchemy/main.py

```diff
@@ -25,19 +25,31 @@
 USE_PYGRAPHVIZ = True
 GRAPHVIZ_AVAILABLE = True
 try:
-    from pygraphviz.agraph import AGraph
+    from pygraphviz.agraph import AGraph # type: ignore
 
     logging.debug("using pygraphviz")
 except ImportError:
     USE_PYGRAPHVIZ = False
     try:
-        from graphviz import Source
+        from graphviz import Source # type: ignore
 
         logging.debug("using graphviz")
     except ImportError:
         logging.error("either pygraphviz or graphviz should be installed")
         GRAPHVIZ_AVAILABLE = False
 
+try:
+    import plantuml  # type: ignore
+    __has_plantuml = True
+
+except ModuleNotFoundError as m:
+    print(m)
```

Comment:
Pleas remove the prints

Reply:
"print function call" is replaced with "logging.debug" call.
I also added "field name"(right/left_col) in link definitions.(models.py:265).